contracts/access/AccessController.sol

```diff
@@ -119,13 +121,8 @@ contract AccessController is IAccessController, GovernableUpgradeable, UUPSUpgra
     /// @param to The address that can be called by the `signer` (currently only modules can be `to`)
     /// @param func The function selector of `to` that can be called by the `signer` on behalf of the `ipAccount`
     /// @param permission The new permission level
-    function setPermission(
-        address ipAccount,
-        address signer,
-        address to,
-        bytes4 func,
-        uint8 permission
-    ) public whenNotPaused {
+    function setPermission(address ipAccount, address signer, address to, bytes4 func, uint8 permission) public {
```

Comment:
Is this function Public now?

Reply:
It wasn't restricted to admin role callers before. I just removed `whenNotPaused`, but I will re introduce pausing in following PR